### PR TITLE
Mphelan/dataverse loader enhance

### PR DESCRIPTION
**Pull request by mikephelan**
dataverse_loader_uri signature refactor to eliminate token argument

### CODE REVIEW

**Comment by mikephelan** [comment]
removed HTTPS scheme, accessing URL params via dict, please review

**Comment by eedeleon** [comment]
PR looks good. A test for the functionality would round it off

**Comment by mikephelan** [comment]
> PR looks good. A test for the functionality would round it off

Proposed testing based on existing pytest examples

**Comment by mikephelan** [comment]
@eedeleon at this point dataverse_uri_loader takes in a dataverse schema URL and requests an HTTP schema URL successfully. I added a test. Please let me know if there is anything further necessary to get this into whitenoise-core. I would like to use it to pull data from opendp-ux.

File: tests/sdk/data/test_dataverse.py [hunk 1]
```diff
@@ -12,8 +12,15 @@
...
+        {"uri":
+         "https://dataverse.harvard.edu/api/access/datafile/:persistentId?persistentId=doi:10.7910/DVN/E9QKIZ/TRLS9U&token=" + os.environ.get('WHITENOISE_DATAVERSE_TEST_TOKEN')}
     ])
 def test_with_doi(dataverse_details):
     df = dataverse_loader(dataverse_details["host"], dataverse_details.get("doi"), dataverse_details.get("token"))
     assert df is not None
+
+def test_uri_only(dataverse_details):
+    df = dataverse_uri_loader(dataverse_details["uri"])
+    assert df is not None
```
Comment (iclrdpsynthdataeval) [inline tests/sdk/data/test_dataverse.py:25]: verify df is a pandas dataframe

**Comment by mikephelan** [comment]
updated test and params for test

**Comment by mikephelan** [comment]
URI-only test now successfully passing, automated tests passing

**Review by eedeleon — APPROVED** [review]
Nice contribution! Looking forward to see more usage of dataverse data with the library

**Pull request merged**